src/Google.Api.Gax/ChannelPool.cs

```diff
@@ -32,11 +32,50 @@ public sealed class ChannelPool
             = new Lazy<Task<ChannelCredentials>>(
                 () => Task.Run(async () => (await GoogleCredential.GetApplicationDefaultAsync()).ToChannelCredentials()));
 
+        private readonly IEnumerable<string> _scopes;
+        private readonly Lazy<Task<ChannelCredentials>> _lazyScopedDefaultChannelCredentials;
+
         // TODO: See if we could use ConcurrentDictionary instead of locking. I suspect the issue would be making an atomic
         // "clear and fetch values" for shutdown.
         private readonly Dictionary<ServiceEndpoint, Channel> _channels = new Dictionary<ServiceEndpoint, Channel>();
         private readonly object _lock = new object();
 
+        // Constructor which exists for ease of testing as much as anything else.
```

Comment:
The comment, or the constructor as well? I was thinking about this, and I wonder whether it would be better to remove it - very little test code is going to create a ChannelPool itself, and we can fix that up.